### ISSUE TITLE
pollInterval should not poll the server when set to zero

### ISSUE THREAD

**Issue by mainawycliffe** [snowballdigital/-flutter-graphql#10]
**Describe the bug**
When `pollInterval` is set to zero, it constantly polls to the server, without stopping. Since it follows Apollo GrapphQL patterns, this should disable polling altogether.

**To Reproduce**

Set `pollInterval` to 0 at QueryOptions

**Expected behavior**
Should disable polling altogether

**Comment by juicycleff** [comment]
A lot of the functionality needs to be tested though. We could have like a swift talk sometime this week and see how to improve all the features in it. Personally, I think we need to outline all the missing feature and unstable ones with react apollo as our guide.

**Comment by mainawycliffe** [comment]
Yeah, I agree. We can use GitHub Projects to keep track of everything.
